scalableminds/webknossos
# Fix TaskType overview graph

The TaskType Overview Graph (http://master.oxalis.at/tasks/overview) does not render correctly because webcola tries to render edges with missing nodes. Which nodes? Each task type is a node. However we also create edges for to so called "future TaskTypes", which are never added a nodes in the first place.

Steps to test:
- Go to http://xyz.oxalis.at/tasks/overview
- Check that graph renders
- (create many Task & TaskTypes so graph is big and beautiful)

Issues:
- fixes #1632 

------


<a href="https://timer.scm.io/repos/537237ce1a00001a003c4a14/issues/1634/create?referer=github" target="_blank">Log Time</a>
